lib/MT/CMS/RebuildTrigger.pm

```diff
@@ -17,7 +17,12 @@ sub config {
     return $app->permission_denied()
         unless $app->can_do('edit_config');
 
-    $param->{default_access_allowed} = $app->config('DefaultAccessAllowed');
+    unless ($param->{default_access_allowed} = $app->config('DefaultAccessAllowed')) {
```

Comment:
This should be something like the following (cf. https://github.com/movabletype/movabletype/blob/2f28bd7b73c31738a011521f87ae71e48e5fc8b4/lib/MT/CMS/User.pm#L699 )
```
$param->{default_access_allowed} = $app->config('DefaultAccessAllowed');
if ( $app->config->is_readonly('DefaultAccessAllowed') ) {
```

Reply:
@charsbar Thank you for your comment! I updated by using `is_readonly` at https://github.com/movabletype/movabletype/pull/1454/commits/91e4f7d15a45a3ddb303403161085a78ebcaff9c.